<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogAddFromYMUrl</class>
 <widget class="QDialog" name="DialogAddFromYMUrl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>925</width>
    <height>638</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Импорт товара из Яндекс.Маркет</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QPushButton" name="pushButtonNext">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../res.qrc">
       <normaloff>:/32/icons/32x32/next.png</normaloff>:/32/icons/32x32/next.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QPushButton" name="pushButtonBack">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../res.qrc">
       <normaloff>:/32/icons/32x32/prev.png</normaloff>:/32/icons/32x32/prev.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QLineEdit" name="lineEditUrl">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="pushButtonParse">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>   Парсинг</string>
     </property>
     <property name="icon">
      <iconset resource="../res.qrc">
       <normaloff>:/32/icons/32x32/1428522468_export.png</normaloff>:/32/icons/32x32/1428522468_export.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="4">
    <widget class="QWebView" name="webViewYandexMarket" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKitWidgets/QWebView</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>lineEditUrl</tabstop>
  <tabstop>pushButtonBack</tabstop>
  <tabstop>pushButtonNext</tabstop>
  <tabstop>pushButtonParse</tabstop>
 </tabstops>
 <resources>
  <include location="../res.qrc"/>
 </resources>
 <connections/>
</ui>
